hack/log/log-dump.sh

```diff
@@ -23,8 +23,10 @@ cd "${REPO_ROOT}" || exit 1
 
 # shellcheck source=hack/ensure-kind.sh
 source "${REPO_ROOT}/hack/ensure-kind.sh"
-# shellcheck source=hack/ensure-kubectl.sh
-source "${REPO_ROOT}/hack/ensure-kubectl.sh"
+
+# installation of kubectl
+KUBECTL="hack/tools/bin/kubectl"
```

Comment:
@sayantani11 it's not finding `make ${KUBECTL}` below because the make target is the full absolute path (TOOLS_BIN uses abspath in the makefile). You can fix it by changing the line above to

```suggestion
KUBECTL=$(realpath hack/tools/bin/kubectl)
```

Reply:
Is it applying to every script using kubectl?

Reply:
And will it be something like this `$(${REPO_ROOT}/hack/tools/bin/kubectl)` or we use real path?

Reply:
@sayantani11 you just need to add $(realpath ...)` where `...` is the path you had before. So if the path was `../hack/tools/bin/kubectl`,  it would become `$(realpath ../hack/tools/bin/kubectl)`.

Same thing in every script.